docs/tech/vumi.md

```diff
@@ -0,0 +1,28 @@
+# Vumi
+
+We build on top of our Vumi platform for SMS, USSD and other messaging protocols.
+
+[Vumi][] is a scalable messaging engine which we use for SMS, USSD and other messaging
+protocols.
+
+We [build Vumi in a container][docker-vumi] and [push it to Docker Hub][dockerhub].
```

Comment:
Could maybe add something like: "Nowadays, for the most part Vumi is only used via Junebug." Except my English is bad right now.